moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -64,7 +64,7 @@ const std::string SETUP_ASSISTANT_FILE = ".setup_assistant";
 // ******************************************************************************************
 ConfigurationFilesWidget::ConfigurationFilesWidget(QWidget* parent,
                                                    moveit_setup_assistant::MoveItConfigDataPtr config_data)
-  : SetupScreenWidget(parent), config_data_(config_data), has_generated_pkg_(false), first_focusGiven_(true)
+  : SetupScreenWidget(parent), config_data_(config_data), has_generated_pkg_(false)
```

Comment:
For this to work correctly -e.g.if the user decides to go back to any screen after navigating to the configuration screen for the first time- , [I had `loadGenFiles()` get called everytime the `foucusGiven()` is called](https://github.com/ros-planning/moveit/pull/1046/files#diff-82144bb4cd694a06a3f4ac970a305343R699).